packaging/suse/rpm/systemd/trento-mcp-server.config.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# Trento MCP Server Configuration File
+# This file contains the configuration settings for the Trento MCP Server.
+# The server acts as a bridge between MCP clients and the Trento API, exposing
+# selected API operations as MCP tools based on the configured filters.
+#
+# Configuration can be provided via this file, environment variables, or command-line flags.
+# Command-line flags take precedence over environment variables, which take precedence over this file.
+
+## The port on which the MCP Server listens for incoming connections.
+port: 8080
+## The transport protocol to use for MCP communication. Options: 'streamable' (default) or 'sse'.
+transport: streamable
+## List of paths to the OpenAPI specification files (local or remote URLs).
+oasPath:
```

Comment:
I'm not sure if I would put these 2 values (at least `trentoURL` and `oasPath`).
The user will always need to change this entries.
What about commenting them with dummy examples?
Otherwise, the user might start the mcp server with this config, It would start, and the user wouldn't even know the values must be changed

Reply:
> I vote for having an .example file or comment out all the values.

Yep, +1. I have just replaced it with an ".example" file, with some `<CHANGE_ME>` values (they are commented out, though)

>In fact, we do these 2 things in trento-web/wanda and the agent 🙈 As trento-agent is written in golang, maybe we could follow that approach.

Ouch, that's true... however, since the MCP Server is (likely) being placed in the same place where TrentoWeb/Wanda is running... I feel the MCP should be more consistent with their conif approach, rather than with the agent. 

For now, I have just pushed some changes, moving the config file from a `yaml` to an `env` format, along with minor improvements (better names for the env vars, etc). Let's see what you think.